Makefile

```diff
@@ -6,7 +6,9 @@ SHELL := /usr/bin/env bash
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.5.0
+VERSION ?= 0.3.0
```

Comment:
should this be 0.8.0 instead?

Reply:
why 0.8.0?

Reply:
This is the version used in the upstream currently: https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/Makefile

Reply:
Right, but here’s what we use in RHODA.
https://github.com/RHEcosystemAppEng/dbaas-operator/blob/release-0.1.5/controllers/reconcilers/constants.go

I’m good with changing though. Just have to pass right version during make

Reply:
Yes, needs to pass in the right version during the build, just as what was done in previous build of RHODA. Changing it to 0.8.0 can make rebasing easier and less confusing.

Reply:
done